spring-integration-stomp/src/test/resources/log4j.properties

```diff
@@ -4,5 +4,6 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d %c{1} [%t] : %m%n
 
-log4j.category.org.springframework.integration=WARN
-log4j.category.org.springframework.integration.stomp=WARN
+log4j.category.org.springframework.messaging=DEBUG
+log4j.category.org.springframework.integration=DEBUG
```

Comment:
Yes. To have more info to investigate those sporadic failure on Bamboo.
See commit comments